samza-core/src/main/java/org/apache/samza/container/grouper/stream/DefaultStreamPartitionMapper.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.samza.container.grouper.stream;
+
+import com.google.common.base.Preconditions;
+import org.apache.samza.Partition;
+import org.apache.samza.system.SystemStreamPartition;
+
+public class DefaultStreamPartitionMapper implements StreamPartitionMapper {
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public SystemStreamPartition getSSPBeforeExpansion(SystemStreamPartition currentSystemStreamPartition, int partitionCountBeforeExpansion, int partitionCountAfterExpansion) {
```

Comment:
Please consider using this implementation as the default interface impl in StreamPartitionMapper